docs/develop/api-reference/index.mdx

```diff
@@ -1,17 +1,20 @@
 ---
 sidebar_position: 5
 Title: API reference information
+sidebar_label: API reference
+description: Learn the API calls you need to know for Momento Serverless Cache.
 ---
 
 import { SdkExamples } from "@site/src/components/SdkExamples";
 
-# API Reference
+# API reference for Momento Serverless Cache
```

Comment:
Reference should be capitalized here

Reply:
This is following AWS style. Since it is not a proper name, a product name, or company name, it should not be capitalized.